### PR TITLE
Bug fixes & Code improvement

### PR DESCRIPTION
**Pull request by sarath-sasi-panickaveetil**
> Added new Support central Logo
> Removed onclick from all div elements
> Updated Reset function
> Added PST datetime in OPM and OPM pages
> Filtercard content issue on selecting 12 AM time values
> Alignment issues on session page

### CODE REVIEW

File: src/views/opmComparison.tsx [hunk 1]
```diff
@@ -473,11 +479,7 @@ const OpmComparison: React.FC = () => {
         </div>
       )}
       {location.pathname.includes("opmcomparison") && (
```
Comment (thasnim-cm) [inline src/views/opmComparison.tsx]: can u import this "opmcomparison" from MENU_LIST of Utils? Or if we change the page url later we will have to update here as well. Same applies for location.pathname.includes("home")

File: src/views/opmComparison.tsx [hunk 2]
```diff
@@ -473,11 +479,7 @@ const OpmComparison: React.FC = () => {
...
-        <div
-          className={`${isLoading ? "h-[80%]" : ""} ${
-            IS_FULLSCREEN ? "" : "sm:mx-4"
-          }`}
-        >
+        <div className={`${isLoading ? "h-[80%]" : ""}`}>
```
Comment (thasnim-cm) [inline src/views/opmComparison.tsx]: change h-[80%] to  h-4/5

Comment (sarath-sasi-panickaveetil) [reply src/views/opmComparison.tsx]: done

File: src/views/opmComparison.tsx (hunk 1 above)
Comment (sarath-sasi-panickaveetil) [reply src/views/opmComparison.tsx]: done

File: src/components/charts/BarChart.tsx [hunk 3]
```diff
@@ -374,29 +379,31 @@ const BarChart = () => {
...
-              </div>
-            )}
-          </div>
+              {!disabled && (
+                <a
+                  onClick={() => resetFormEntry()}
+                  className="text-gray-300 font-normal text-xs ml-2 cursor-pointer"
+                >
+                  {LABELS.reset}
+                </a>
```
Comment (AdithyaShaj) [inline src/components/charts/BarChart.tsx]: Use button instead of a tag. a tag expects an internal or external redirection

**Review by thasnim-cm — APPROVED** [review]

**Pull request merged**